src/bin/find_files_to_check.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+
+# A POSIX variable
+OPTIND=1         # Reset in case getopts has been used previously in the shell.
+
+##### Configuration #####
+
+blacklistedDirectories=(
+".git"
+".idea"
+".vagrant"
+"vendor"
+)
+
+##### Variables #####
+
+scriptDirectory=""
+findParentWithFile=""
+rootDirectory=""
+targetBranch=""
+stopword=""
```

Comment:
warum ist stopword setzbar? sollten wir das nicht festlegen?

Reply:
Das wird in dem Script des jeweiligen Analysetools gesetzt und hier als parameter übergeben. also find_files_to_check sucht für sniff_php nach stopword .dontSniffPHP